neural_network/activation_functions/leaky_rectified_linear_unit.py

```diff
@@ -0,0 +1,47 @@
+"""
+Leaky Rectified Linear Unit (LeakyReLU)
+
+Input: vector (type: np.ndarray) , alpha (type: float)
+Output: vector (type: np.ndarray)
+
+UseCase: LeakyReLU solves the issue of dead neurons or vanishing gradient problem.
+Refer the below link for more information:
+https://en.wikipedia.org/wiki/Rectifier_(neural_networks)#Leaky_ReLU
```

Comment:
Minor grammar/spelling fixes
```suggestion
Leaky Rectified Linear Unit (Leaky ReLU)

Use Case: Leaky ReLU solves the issue of dead neurons, or the vanishing gradient problem.
Refer the below link for more information:
https://en.wikipedia.org/wiki/Rectifier_(neural_networks)#Leaky_ReLU
```